examples/register_and_change_username.php

```diff
@@ -32,25 +34,30 @@
 $commandMap = [
     RegisterUser::class => [
         'handler' => function (array $state, Message $message) use (&$factories): AggregateResult {
-            return \Prooph\MicroExample\Model\User\registerUser($state, $message, $factories['emailGuard']());
+            return User\registerUser($state, $message, $factories['emailGuard']());
         },
         'definition' => UserAggregateDefinition::class,
     ],
+    'handler' => function (array $state, Message $message): AggregateResult {
```

Comment:
First of all, this key is expected to be a command name. Second the aggregate definition is missing. Third the function `Prooph\MicroDo\UserWrite\Model\User\registerWithData` does not exist. So what's the point?